### ISSUE TITLE
Flowrun Timeline Scaling 

### ISSUE THREAD

**Issue by prefectcboyd** [PrefectHQ/prefect-ui-library#1722]
Flowruns that run into hours, appear to hit a visualization issue with the Y-axis on the timeline.
This was customer reported, and I have not reproduced, but can attempt if need be for better details.
It seems like the "100%" or the top of the axis, is some rounding factor .
In the image below, you can see at the 25% line, it counts very specifically 41m, with no run correlated at that time.
Then we see at 50% a 1h tick, which seems normal.
Then we see a 2h line - which isn't unusual, as it could be logarithmic scaling to keep displaying more items with larger scale, but then the last line is also 2h. This indicates to me the first "2h" is really a rounding factor of double the 41m, while the second 2h (the 100% line) is the actual 2h mark (which is double the 1h mark).

All this to say - I understand __what__ is being presented here, but when looking for a quick snapshot view of the timeline, its not quickly evident or intuitive what is being displayed. 

In short - I think the scaling is fine and the issue  is with the rounding to nearby neighbors (2h -> 2h)
![image](https://github.com/PrefectHQ/prefect-ui-library/assets/105737359/9ad78197-86b1-4bac-aab9-d1c49b15514a)
